src/database/helpers/checkLanguage.ts

```diff
@@ -0,0 +1,17 @@
+import { UserInputError } from 'apollo-server';
+import config from '../../config';
+
+/**
+ * Checks if the language code passed in is in the list of allowed languages.
+ *
+ * @param language
+ */
+export const checkLanguage = (language: string): void => {
+  if (!config.app.languages.includes(language)) {
```

Comment:
I think `.includes()` is case sensitive but I'm not sure. If it is then see the comment below

Reply:
Good point @Herraj, I was making the updates in different areas of the code without thinking what's going to be executed in which order 🙃. I'll remove those `'toUpperCase()` calls as they should never be needed.